src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -535,19 +535,19 @@ void EnGinkoMan_SetupBankAward2(EnGinkoMan* this) {
     this->actionFunc = EnGinkoMan_BankAward2;
 }
 
-// separate function to handle bank rewards... if the bank has a parent actor? might be unused
+// separate function to handle bank rewards... called while the player is receiving the award
 void EnGinkoMan_BankAward2(EnGinkoMan* this, GlobalContext* globalCtx) {
     if (Actor_ProcessTalkRequest(&this->actor, &globalCtx->state)) {
         if (!(gSaveContext.weekEventReg[10] & 8) && (this->curTextId == 0x45B)) {
             // "What's this? You've already saved up 200 Rupees!?!  Well, little guy, here's your special gift. Take
             // it!"
```

Comment:
This should also be a summary
```suggestion
            // Player has saved up 200 Rupees
```

Reply:
This is not something I changed. There are many other comments in this file that have the exact text, but the purpose behind this pull request was not to fix this, but to clear up misunderstandings about the parent actor and the incorrect message quote.

Additionally, there doesn't seem to be a consensus around whether the direct quotes should be here or not. If they shouldn't be here, anyone else is free to make a pull request to change them.

Reply:
The real issue is that the file really should have been viewed by more eyes before it was accepted, and by making your changes, you've caused people to look more closely at it. This PR is a small step in the right direction, but personally I would do every change you've made differently. Thinking on it more and considering you seem to not want to spend more time on this file (which is perfectly fine), I guess I'm OK with your changes as is.